inference-engine/tests/functional/plugin/myriad/shared_tests_instances/single_layer_tests/gather_nd.cpp

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "single_layer_tests/gather_nd.hpp"
+#include <vpu/private_plugin_config.hpp>
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+
+const std::vector<InferenceEngine::Precision> netPrecisions = {
+    InferenceEngine::Precision::FP16,
+};
+
+const std::vector<InferenceEngine::Precision> indicesPrecisions = {
+    InferenceEngine::Precision::I32,
+};
+
+const std::vector<gatherNDLayerParams> layerParams = {
+    // ngraph examples
+    // N = 1000: Not enough memory. replaced with 500
+    // Probably calculating with FP32 instead of FP16
+    {0, {500, 256, 10, 15}, {25, 125, 3}},
+    {2, {30, 2, 100, 35}, {30, 2, 3, 1}},
+    // some random tests
+    {0, {3, 3}, {2, 2}},
+    {0, {5, 3}, {2, 1}},
+    {0, {5, 3, 4}, {2, 2}},
+    {0, {6, 3, 4}, {2, 1, 2}},
+    {0, {5, 2, 6, 8}, {1}},
+    {0, {6, 6, 9, 7}, {2}},
+    {0, {2, 4, 9, 4}, {3}},
+    {0, {5, 2, 3, 7}, {4}},
+    {1, {2, 2, 2}, {2, 1}},
+    {1, {2, 2, 2, 2}, {2, 1}},
+    {2, {2, 2, 2, 2}, {2, 2, 1}},
+};
+
+using Config = std::map<std::string, std::string>;
+
+INSTANTIATE_TEST_CASE_P(
+    smoke_GatherND,
+    GatherNDLayerTest,
+    testing::Combine(
+        testing::ValuesIn(layerParams),
+        testing::ValuesIn(netPrecisions),
+        testing::ValuesIn(indicesPrecisions),
+        testing::Values(InferenceEngine::Layout::ANY),
+        testing::Values(InferenceEngine::Layout::ANY),
```

Comment:
Are Layouts parameters really needed?

Reply:
This functionality was added by KMB request